sdk/identity/azure-identity/CMakeLists.txt

```diff
@@ -63,6 +77,7 @@ target_include_directories(
   azure-identity
     PUBLIC
       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
+      $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/sdk/core/azure-core/inc>
```

Comment:
This is confusing.  Are we getting the azure-core src code into sdk/service/package path?   We should use the build folder to download the src for building.